cookbook/doctrine/index.rst

```diff
@@ -14,4 +14,6 @@ Doctrine
     resolve_target_entity
     mapping_model_classes
     registration_form
+    pdo_session_storage
+    mongodb_session_storage
```

Comment:
this look weird to me. These are not about Doctrine but about native PDO and MongoDB